hasktorch/src/Torch/Typed/NN/Recurrent/LSTM.hs

```diff
@@ -200,14 +200,14 @@ instance A.Parameterized (LSTMLayer inputSize hiddenSize directionality dtype de
             (UnsafeMkParameter bh')
             )
 
-data LSTMLayerStackSpec
-  (inputSize :: Nat)
-  (hiddenSize :: Nat)
-  (numLayers :: Nat)
-  (directionality :: RNNDirectionality)
-  (dtype :: D.DType)
-  (device :: (D.DeviceType, Nat))
-  = LSTMLayerStackSpec deriving (Show, Eq)
+data LSTMLayerStackSpec (inputSize :: Nat) (hiddenSize ::
+                                         Nat) (numLayers :: Nat) (directionality ::
+                                                                    RNNDirectionality) (dtype ::
+                                                                                          D.DType) (device
+                                                                                                      ::
+                                                                                                      (D.DeviceType,
+                                                                                                       Nat)) = LSTMLayerStackSpec
+    deriving (Show, Eq)
 
```

Comment:
Can we arrange the lines?

Reply:
Yes, this looks horrible :(

Reply:
oops my bad, sometimes it does crazy stuff didn't spot this. Will change

Reply:
Is this the ormolu default? do you have to manually change it back per instance or is there a flag that can keep it from doing this?